BackgroundTask.py

```diff
@@ -280,7 +279,7 @@ def queryEdsm(self):
             edsmJson = json.loads(response)
             return edsmJson["id64"], len(edsmJson["bodies"])
         except:
-            if __debug__: print_exc()
+            print_exc()

```

Comment:
Should also be in debug. But I noticed a rather large stack trace when that check fails.
Maybe we should change it to a single line: "Failed to get information about bodies in {systemname} from EDSM``